easybuild/easyconfigs/p/Pillow/Pillow-3.2.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow'
+version = '3.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pillow.readthedocs.org/'
+description = """Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
+ PIL is the Python Imaging Library by Fredrik Lundh and Contributors."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('libjpeg-turbo', '1.4.2','-NASM-2.12.01'),
+    ('libpng', '1.6.21'),
+    ('zlib', '1.2.8'),
+    ('LibTIFF', '4.0.6'),
+    ('freetype', '2.6.3'),
```

Comment:
OK, the issue is basically that `matplotlib` 1.5.1 uses `freetype` 2.6.2 as a dep, while you are using `freetype` 2.6.3 as a dep here...

See also:

```
$ eb --check-conflicts scikit-image-0.12.3-foss-2016a-Python-2.7.11.eb
== temporary log file in case of crash /var/folders/xl/p7lmsxcs0bsg5_zv9y9lzk5c0000gn/T/eb-GdzpLC/easybuild-tAhSFM.log
Conflict found for dependencies of scikit-image-0.12.3-foss-2016a-Python-2.7.11: freetype-2.6.2-foss-2016a vs freetype-2.6.3-foss-2016a 
	freetype-2.6.2-foss-2016a as dep of: matplotlib-1.5.1-foss-2016a-Python-2.7.11, scikit-image-0.12.3-foss-2016a-Python-2.7.11
	freetype-2.6.3-foss-2016a as dep of: Pillow-3.2.0-foss-2016a-Python-2.7.11, scikit-image-0.12.3-foss-2016a-Python-2.7.11
Conflict between (dependencies of) easyconfigs: freetype-2.6.2-foss-2016a vs freetype-2.6.3-foss-2016a 
	freetype-2.6.2-foss-2016a as dep of: matplotlib-1.5.1-foss-2016a-Python-2.7.11, scikit-image-0.12.3-foss-2016a-Python-2.7.11
	freetype-2.6.3-foss-2016a as dep of: Pillow-3.2.0-foss-2016a-Python-2.7.11, scikit-image-0.12.3-foss-2016a-Python-2.7.11
```

The output of the tests isn't clear though, I'll try and get that fixed.